webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -14083,7 +14107,13 @@ var userOverride = {
                     );
                     $mck_file_input.trigger('click');
                 });
-
+                $mck_img_upload.on('click', function (e) {
+                    e.preventDefault();
+                    // kmWidgetEvents.eventTracking(
```

Comment:
@devashishmamgain 
- Shall we add event tracking for this like we have added for the othe buttons?

Reply:
yes